planner/core/stringer.go

```diff
@@ -191,6 +191,18 @@ func toString(in Plan, strs []string, idxs []int) ([]string, []int) {
 			r := x.InnerJoinKeys[i]
 			str += fmt.Sprintf("(%s,%s)", l, r)
 		}
+	case *PhysicalIndexMergeJoin:
+		last := len(idxs) - 1
+		idx := idxs[last]
+		children := strs[idx:]
+		strs = strs[:idx]
+		idxs = idxs[:last]
+		str = "IndexMergeJoin{" + strings.Join(children, "->") + "}"
+		for i := range x.OuterJoinKeys {
+			l := x.OuterJoinKeys[i]
+			r := x.InnerJoinKeys[i]
+			str += fmt.Sprintf("(%s,%s)", l, r)
```

Comment:
No need to use `fmt.Sprintf()`, it's very expensive.

Reply:
`ToString` is only used for the test. Maybe `fmt.Sprintf()` is ok? BTW, it's consistent as other physical plans.